packages/graphql-yoga/src/server.ts

```diff
@@ -475,6 +476,32 @@ export class YogaServer<
         this.logger.debug(`Processing GraphQL Parameters done.`);
       }
 
+      /** Ensure that error thrown from subscribe is sent to client */
+      if (result && Symbol.asyncIterator in result) {
```

Comment:
You don't use `isAsyncResult` util we have ?

Reply:
The typing of this function is incorrect result can never be async iterable, if we want to do this change we need to refactor and fix the typings (which I will do ofc).